src/settings/tabenvironment.cpp

```diff
@@ -0,0 +1,99 @@
+#include "tabenvironment.h"
+
+#include <QDesktopServices>
+#include <QVBoxLayout>
+#include <QSettings>
+#include <QPushButton>
+#include <QComboBox>
+#include <QLabel>
+#include <QDebug>
+#include <QUrl>
+#include <QDir>
+
+TabEnvironment::TabEnvironment(QWidget *parent)
+    : QWidget(parent)
+{
+    QVBoxLayout *mainLayout = new QVBoxLayout;//(this);
+    mainLayout->addStretch(mBasicStretch);
+    mainLayout->setSpacing(mSpacing);
+
+    // getting data from QSettings
+    QSettings s;
+    mStyleCurrernt = (s.contains("style") ? s.value("style").toString() : mStyleDefault);
+    mStyleNew = mStyleCurrernt;
+    QString userName = (s.contains("userName") ? s.value("userName").toString() : "");
+
+    //      STYLE
+    // set a label
+    QLabel *editorStyleLabel = new QLabel(tr("Application style"), this);
+    mpComboEditorStyle = new QComboBox(this);
+    mpComboEditorStyle->setMaximumWidth(mEditorStyleWidth);
+    mpComboEditorStyle->setMinimumWidth(mEditorStyleWidth);
+    mpComboEditorStyle->addItems(mEditorStylesList);
+    mainLayout->addWidget(editorStyleLabel);
+    mainLayout->addWidget(mpComboEditorStyle);
+    mainLayout->addSpacing(mRowSpasing);
+
+    // set combobox acording to settings
+    mpComboEditorStyle->setCurrentIndex(0);
+    for(int i = 0; i < mEditorStylesList.size(); ++i)
```

Comment:
this &item again

Reply:
according code convention for local variables in loops it's possible name

Reply:
I didn't mean code convention style. I meant that you don't need to create this variable. You can rewrite this code without using "&item". 
`for(int i = 0; i < mStandartList.size(); ++i)
{
    if ( mStandartList[i] == mStandartCurrernt)
    {
        mpComboStandart->setCurrentIndex(i);`
        break;
    }
}`

Reply:
I got it